engine/apps/auth_token/models/user_schedule_export_auth_token.py

```diff
@@ -31,6 +30,22 @@ def create_auth_token(cls, user: User, organization: Organization) -> Tuple["Use
             user=user,
             organization=organization,
         )
-        description = "User schedule export token was created by user {0}".format(user.username)
-        create_organization_log(organization, user, OrganizationLogType.TYPE_SCHEDULE_EXPORT_TOKEN_CREATED, description)
         return instance, token_string
+
+    # Insight logs
+    @property
+    def insight_logs_type_verbal(self):
+        return "user_schedule_export_token"
+
+    @property
+    def insight_logs_verbal(self):
+        return f"Users chedule export token for {self.user.username}"
+
+    @property
+    def insight_logs_serialized(self):
+        # Schedule export tokens are not modifiable now, so return empty dict to implement InsightLoggable interface
```

Comment:
I'm not sure they will be modifyable any time. They are just got recreated

Reply:
Anyway we need to implement this method to implement InsightLoggable interface